samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitor.java

```diff
@@ -83,9 +83,10 @@ public void monitor() throws Exception {
                              String.format("%s-%s", jobInstance.getJobName(), jobInstance.getJobId()));
       try {
         JobStatus jobStatus = jobsClient.getJobStatus(jobInstance);
+        LOG.info("Job: {} has the status: {}.", jobInstance, jobStatus);
         for (Task task : jobsClient.getTasks(jobInstance)) {
+          LOG.info("  Evaluating stores for job: ({}) task: ({})", jobInstance, task);
```

Comment:
I'm guessing this indentation is for readability in output? If so, maybe use \t if that works, or leave a comment saying so (so that it doesn't get removed in a refactor).

Remove parens around job and task name for consistency. @